app.js

```diff
@@ -7,15 +7,58 @@ client.on('ready', () => {
 });
 client.login(process.env.BOT_TOKEN)
 
+async function setup(msg) {
```

Comment:
this shouldn't be `async`
the keyword `async` is reserved for special things like making an API call, because it's like its own process which takes some time. This function just does simple JS operations, so we should remove `async`

Reply:
Did some further reading on that and it makes sense! Weird that the guides I followed had async in there.